times_test.go

```diff
@@ -64,8 +64,42 @@ func TestFuture(t *testing.T) {
 func TestRange(t *testing.T) {
 	start := time.Time{}
 	end := time.Unix(math.MaxInt64, math.MaxInt64)
-	x := RelTime(start, end, "ago", "from now")
+	x := RelTime(start, end, "ago", "from now", default_magnitudes)
```

Comment:
Idiomatic Go style suggests this identifier name should take the form of `defaultMagnitudes` rather than using an underscore as a separator. See https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps.